### PR TITLE
[4.10] remove useless priority field in ci job name

### PR DESCRIPTION
**Pull request by jianlinliu**

### CODE REVIEW

**Comment by jianlinliu** [comment]
@liangxia @jhou1 per our discussion in slack, now the latest 4.10.z get released, so it is time to remove the priority field in ci job name. When you have time, pls help review. cc @gpei @yunjiang29

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10-periodics.yaml [hunk 1]
```diff
@@ -6769,15 +6769,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.10"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-aws-ipi-p3-f28-long-duration-part3-3
-  reporter_config:
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10-periodics.yaml:6773]: Several `reporter_config` are removed unexpectedly.

Comment (jianlinliu) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10-periodics.yaml:6773]: fixed.

**Pull request merged**